ibis/expr/operations.py

```diff
@@ -871,6 +871,14 @@ class Arbitrary(Reduction):
     output_type = rlz.scalar_like('arg')
 
 
+class BitXor(Reduction):
```

Comment:
hmm do we have any other bit operations?

Reply:
`bit_and` and `bit_or` are also available, though `bit_xor` is the only one suited for my checksum usecase. I could see `bit_or` being useful for a binary flags column, though.

Reply:
I've pushed a commit adding `bit_and` and `bit_or` methods.